docs/fields.md

```diff
@@ -1,6 +1,6 @@
 # Kustomization File Fields
 
-An explanation of the fields in a [kustomization.yaml](glossary.md#kustomization) file.
+An explanation of the fields in a [kustomization.yaml/json](glossary.md#kustomization) file.
```

Comment:
Let's not add kustomization.json here since all the following fields are in yaml format.